package.json

```diff
@@ -48,6 +48,7 @@
     "scrollmonitor": "^1.2.4",
     "stylus": "^0.54.8",
     "stylus-loader": "^3.0.2",
+    "swiper": "^7.3.0",
```

Comment:
That's not super light
![image](https://user-images.githubusercontent.com/16245250/143307748-53cbe610-2155-472e-b28c-3086242c9d3a.png)